src/components/DataView/DataViewColumnDrawerContent.tsx

```diff
@@ -1,165 +1,186 @@
 import React, { useState } from "react";
 import styled from "styled-components";
-import ArrowDownwardIcon from "@mui/icons-material/ArrowDownward";
-import ArrowUpwardIcon from "@mui/icons-material/ArrowUpward";
-
-import Button from "../Button";
+import DragIndicatorIcon from "@mui/icons-material/DragIndicator";
 import theme from "@root/theme";
 import CheckboxList from "@root/components/CheckboxList";
-import DrawerContent from "@root/components/DrawerContent";
 import { useMosaicTranslation } from "@root/i18n";
+import DrawerHeader from "../DrawerHeader";
+import {
+	DragDropContext,
+	Draggable,
+	Droppable,
+	DropResult,
+} from "react-beautiful-dnd";
+import { DataViewProps } from "./DataViewTypes";
+import { ButtonProps } from "../Button";
+
+const ColumnTitle = styled.h2`

```

Comment:
Please let's move all parts of this component to their own folder (styled components, types, index, etc).